ui/tests/acceptance/secrets/backend/pki/list-test.js

```diff
@@ -22,6 +22,7 @@ module('Acceptance | secrets/pki/list', function(hooks) {
     await mountAndNav(assert);
     assert.equal(currentRouteName(), 'vault.cluster.secrets.backend.list-root', 'redirects from the index');
     assert.ok(page.createIsPresent, 'create button is present');
+    await click('[data-test-configuration-tab]');
```

Comment:
Added these because the Configure link is now only shown when you are viewing the Configuration tab